tests/integration/it_utils.py

```diff
@@ -60,6 +65,20 @@
 LEDGER_ACCEPT_REQUEST = GenericRequest(method="ledger_accept")
 
 
+class AsyncTestTimer:
```

Comment:
Why can't the normal timer be used in both situations?

Reply:
AsyncTestTimer works for Async callback functions - sync functions don't work because asyncio.ensure_future (event loop error stuff)

I originally thought SyncTestTimer would work for everything (and it kinda did?). It worked in that it passed all test cases. However, I encountered a weird bug where on AsyncWebsocketClient (only), the client.request call never finished so the poetry poe test didn't exit process (which is why report coverage never called after and the CI tests ran indefinitely). As a result, I made a new test timer implementation for async tests.